region/maridia/inner-green.json

```diff
@@ -1304,6 +1365,21 @@
               ],
               "openEnd": 1
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Evir Projectile",
+                  "notable": false,
+                  "requires": [ "h_canNavigateUnderwater" ],
+                  "note": [
+                    "Locate the position where Samus needs to be to barely trigger the Evir projectiles.",
+                    "Crouch just outside of this zone, turn around into it and then back and quickly get to the door. There is a 3 pixel starting window that works suitless."
```

Comment:
Moonwalk works really well

Reply:
Walk into the door shell.  Wait for the Evir to be able to shoot again.  Then turn right and left and run out.